tools/x-docs/src/docs/guides/components/cookbook/styling.md

```diff
@@ -1,7 +1,94 @@
 # Styling
 
-Currently, there's no officially-supported way of providing styling for components. To begin with, all our components that require styling are built to be compatible with Origami styles, and the recommended way of using these is by including the CSS from the [Origami Build Service](https://www.ft.com/__origami/service/build/v2/) and referencing the classes in your markup.
+## Origami components
 
-For styles specific to a single x-dash component, we suggest writing a CSS file and including it in the package, then importing it into your app however you normally include styles (e.g. using a SCSS `@import`).
+Some x-dash components, such as [x-teaser](/components/x-teaser/readme.md), are built to be compatible with styles provided by Origami components, by using their classnames. These styles ase expected to be provided by the application that's consuming the x-dash component, for example by including the CSS from the [Origami Build Service](https://www.ft.com/__origami/service/build/v2/). To include these styles in a component's demos in the documentation and component explorer, use the `dependencies` object in your component's [`stories/index.js`](/tools/x-docs/guides/components/setup.md#indexjs).
 
-We're actively investigating better-integrated ways of including styling with components.
+## Component-specific styles
+
+Some components may require styling not provided by existing Origami components. If your component's styles might be useful outside of the FT.com ecosystem, you may want to speak to the [Origami team](http://origami.ft.com/) about creating its styles as an Origami component.
+
+If your component will only be used within FT.com, you can include its styles as CSS Modules within the component.
+
+### CSS Modules
+
+A CSS Module is a self-contained CSS file that can be used with ECMAScript `import`. When a component is bundled, its CSS Modules are bundled into a single `.css` file, and their classes are obfuscated into effectively-random strings, so styles from a component cannot interfere with any other part of a page they're included into. The `default` export of a CSS Module is an object containing its classes as keys, and their obfuscated versions as values. This allows you to reference the classes from your component by their readable names, and output the obfuscated names when the component is used.
```

Comment:
"When a component is bundled" => "When a component is built"
"effectively-random" => "random"
"its classes" => "its class names"

This is a good explanation 👍

Reply:
the mathematician in me cannot bear to write "random" for this 😬

it's based on a hash of the rule contents

it's high entropy but it's not random

Reply:
"obfuscated into non-human readable strings" ?